.github/workflows/pytest.yml

```diff
@@ -34,19 +36,22 @@ jobs:
 
       - uses: actions/cache@v2
         if: startsWith(runner.os, 'Windows')
+        id: cache_Windows
         with:
           path: ~\AppData\Local\pip\Cache
           key: ${{ runner.os }}-pip-${{ hashFiles('setup.cfg') }}
           restore-keys: |
             ${{ runner.os }}-pip-
 
       - name: pip installs
+        if: steps.cache_${{ runner.os }}.outputs.cache-hit != 'true'
         run: |
-          pip install pytest pytest-cov pytest-xdist setuptools_scm
-          pip install -e .
+          pip install -e .[test]
```

Comment:
How to I interpret this? we don't perform an install when the pip cache gets loaded? why?

Reply:
you're right, it doesn't make any sense. I saw this pattern on Stack Overflow, where somebody complained that pip is still downloading things where it should use the cache.